commit baac03018283296f90c08366e0e092c95d25173b
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Tue Oct 18 14:34:34 2022 +0200

    cuda_gpu_multidim_array.cu is just an include file containing templates.
    Needs to be before cuda_gpu_geo_transformer.cu

diff --git xmipp-3.22.07-Helios/src/xmipp/libraries/reconstruction_cuda/cuda_all.cpp xmipp-3.22.07-Helios/src/xmipp/libraries/reconstruction_cuda/cuda_all.cpp
index f7e5178c..8ccbefd3 100644
--- xmipp-3.22.07-Helios/src/xmipp/libraries/reconstruction_cuda/cuda_all.cpp
+++ xmipp-3.22.07-Helios/src/xmipp/libraries/reconstruction_cuda/cuda_all.cpp
@@ -31,6 +31,6 @@
  */
 
 #include "cuda_gpu_bilib.cu"
+#include "cuda_gpu_multidim_array.cu"
 #include "cuda_gpu_geo_transformer.cu"
 #include "cuda_gpu_iirconvolve.cu"
-#include "cuda_gpu_multidim_array.cu"
